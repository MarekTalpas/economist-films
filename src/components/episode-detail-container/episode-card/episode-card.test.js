// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import EpisodeCard from './episode-card';

test('EpisodeCard renders correctly', () => {
  const tree : string = renderer.create(<EpisodeCard />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('EpisodeCard renders with data', () => {
  const tree : string = renderer.create(<EpisodeCard url="http::ptth" />).toJSON();
  expect(tree).toMatchSnapshot();
});