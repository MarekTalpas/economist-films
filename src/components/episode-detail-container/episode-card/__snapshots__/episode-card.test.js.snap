// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EpisodeCard renders correctly 1`] = `
<div
  className="episode-card"
  style={
    Object {
      "background": "#000 url(undefined) no-repeat center center ",
      "backgroundSize": "cover",
    }
  }
/>
`;

exports[`EpisodeCard renders with data 1`] = `
<div
  className="episode-card"
  style={
    Object {
      "background": "#000 url(http::ptth) no-repeat center center ",
      "backgroundSize": "cover",
    }
  }
/>
`;
